src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -0,0 +1,271 @@
+import * as React from 'react';
+import { AlterationEnrichment, ExpressionEnrichment } from "shared/api/generated/CBioPortalAPIInternal";
+import { toConditionalPrecision } from "shared/lib/NumberUtils";
+import styles from "./styles.module.scss";
+import { AlterationEnrichmentRow } from 'shared/model/AlterationEnrichmentRow';
+import { ExpressionEnrichmentRow } from 'shared/model/ExpressionEnrichmentRow';
+import { tsvFormat } from 'd3-dsv';
+import { ExtendedAlteration } from 'pages/resultsView/ResultsViewPageStore';
+import { BoxPlotModel, calculateBoxPlotModel } from 'shared/lib/boxPlotUtils';
+import { NumericGeneMolecularData } from 'shared/api/generated/CBioPortalAPI';
+
+const LOG_VALUE = "LOG-VALUE";
+const LOG2_VALUE = "LOG2-VALUE";
+
+export function calculateAlterationTendency(logOddsRatio: number): string {
+    return logOddsRatio > 0 ? "Co-occurrence" : "Mutual exclusivity";
+}
+
+export function calculateExpressionTendency(logOddsRatio: number): string {
+    return logOddsRatio > 0 ? "Over-expressed" : "Under-expressed";
+}
+
+export function formatLogOddsRatio(logOddsRatio: number): string {
+
+    if (logOddsRatio < -10) {
+        return "<-10";
+    } else if (logOddsRatio > 10) {
+        return ">10";
+    }
+    return logOddsRatio.toFixed(2);
+}
+
+export function formatLogOddsRatioWithStyle(logOddsRatio: number): JSX.Element {
+
+    if (logOddsRatio >= 0) {
+        return <span style={{ color: "green" }}>{formatLogOddsRatio(logOddsRatio)}</span>;
+    } else {
+        return <span style={{ color: "red" }}>{formatLogOddsRatio(logOddsRatio)}</span>;
+    }
+}
+
+export function formatValueWithStyle(value: number): JSX.Element {
+
```

Comment:
i've started using a library called http://numeraljs.com/, which has some nice formatting/precision features.  not merged yet, but you could install it

Reply:
but you don't have to